src/ansys/geometry/core/math/matrix.py

```diff
@@ -129,3 +134,40 @@ def __new__(cls, input: np.ndarray | RealSequence | Matrix = DEFAULT_MATRIX44):
             raise ValueError("Matrix44 should only be a 2D array of shape (4,4).")
 
         return obj
+
+    @classmethod
+    def create_matrix_from_rotation(
+        cls, direction_x: "Vector3D", direction_y: "Vector3D"
+    ) -> "Matrix44":
+        """Matrix44 helper method -- create_matrix_from_rotation."""
+        matrix = cls(
+            [
+                [direction_x.x, direction_x.y, direction_x.z, 0],
+                [direction_y.x, direction_y.y, direction_y.z, 0],
+                [0, 0, 1, 0],
+                [0, 0, 0, 1],
+            ]
+        )
+        return matrix
+
+    @classmethod
+    def create_matrix_from_translation(cls, origin: "Point3D") -> "Matrix44":
+        """Matrix44 helper method -- create_matrix_from_translation."""
```

Comment:
A translation should receive as input a Vector in 3D space, not a Point. You have to modify the input and the implementation accordingly.

Reply:
@jrwest440  do you have any comment on this?